src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -164,7 +164,7 @@ async function getAdditionalFacilityInfo(futureAppointments, useV2 = false) {
 function getAdditionalFacilityInfoV2(appointments) {
   // Facility information included with v2 appointment api call.
   return appointments
-    ?.map(appt => (appt.vaos.facilityData ? appt.vaos.facilityData : null))
+    ?.map(appt => appt?.vaos?.facilityData ?? null)
```

Comment:
Question. What's this line doing? if 'facilityData' return 'facilityData' else return false?

Reply:
@vbahinwillit (??) returns the value to its left if it is NOT null or undefined. Otherwise it returns the value to its right. In this case it will be null.